### PR TITLE
Using recommend video default and fixed small typo optionSet.cpp, gui.cpp

### PR DESCRIPTION
**Pull request by zDEFz**
Using recommend defaults for Video now (100,1000)

FIxing typo "sensetive" changing to "sensitive"

### CODE REVIEW

File: optionSet.cpp [hunk 1]
```diff
@@ -88,9 +88,11 @@ short optionSet::loadOptions() {
 
 		repeatDelayDown = sf::milliseconds(20);
 		repeatSpeedDown = sf::milliseconds(20);
-
-		frameDelay = sf::milliseconds(10);
-		inputDelay = sf::milliseconds(5);
+
+		//frame delay multiplicator 10 = 100 FPS
+		frameDelay = sf::milliseconds(10);
+        //om[it delay multiplicator 1000 = 1000ms
```
Comment (zDEFz) [inline optionSet.cpp]: should be   //input delay multiplicator 1000 = 1000 microseconds - 1ms

**Comment by zDEFz** [comment]
in optionSet.cpp comment for input delay should be
 //input delay multiplicator 1000 = 1000 microseconds - 1ms

why can't i edit the comments in the commit?

**Comment by kroyee** [comment]
You can never edit a commit, for the sake of keeping history.
You can however use git commit --amend to "overwrite" the last commit if you find a small mistake that need to be adjusted or changing the message for example.

**Comment by zDEFz** [comment]
Oh it seems to work - scroll down on files changed and it did change the correct file 👍

**Comment by kroyee** [comment]
Can you adjust the commit so it doesn't show the whole file as deleted and replaced?
I can't actually see what/if anything has been changed in optionSet.cpp.

**Comment by zDEFz** [comment]
How to?

the changes in the files was
 +       //frame delay multiplicator 10 = 100 FPS
 +		frameDelay = sf::milliseconds(10);
 +        //input delay multiplicator 1000 = 1000 microseconds - 1ms
 +		inputDelay = sf::milliseconds(1);
 +

**Comment by zDEFz** [comment]
Sry im a git newb

**Comment by kroyee** [comment]
You and me both!
I'm not sure, but I suspect that there is a small change somewhere in the beginning of the file and it is somehow confusing git.

**Comment by kroyee** [comment]
Seems to be an issue with different programs using different characters to mark newline.
Take a look here: http://stackoverflow.com/questions/1889559/git-diff-to-ignore-m

**Comment by kroyee** [comment]
It was fine for the first file though... Did you edit it in a different way second time?

**Comment by zDEFz** [comment]
First i just copy and pasted file contents but it didn't copy the comments.

Then i just made changes to the folder and used a git bash in that folder.
git commit --amend and then save with shift and ::wq

This didn't help.
Then i did close the pull request and made a new.

I have my own fork - comparing mine to yours gives me the ability to make a pull request.



Do you see the new pull request? I just did it with github desktop and it worked fine.

**Pull request closed without merging**